marlowe-dashboard-client/src/Template/Types.purs

```diff
@@ -59,14 +59,14 @@ instance inputFieldErrorRoleError :: InputFieldError RoleError where
 
 data SlotError
   = EmptySlot
-  | PastSlot
+  | NegativeSlot
```

Comment:
Is this value being used? The `slotError` function doesn't have `PastSlot` anymore and a quick search only show usage in this file.

Reply:
No, it's a placeholder for the future. I've added a comment in the `State` module explaining that we're not validating slot inputs yet (because they're readonly). I'm guessing that when we do validate them, we'll want to rule out negative integers - and not past slots (as I'd previously thought), because they'll be relativised to the start of the contract.